src/ExaTioga.cpp

```diff
@@ -115,6 +115,7 @@ void ExaTioga::perform_connectivity()
     {
         auto tmon = tioga_nalu::get_timer("tioga::performConnectivity");
         amrex::Print() << "TIOGA unstructured connectivity" << std::endl;
+	m_tioga.assembleComms();
```

Comment:
We should move this to after this - https://github.com/Exawind/tioga_utils/blob/exawind/src/ExaTioga.cpp#L40

Reply:
I think (?) `assembleComms` should be called _in addition_ to `setCommunicator`. They look like they are doing very different things.

Reply:
that is correct

Reply:
`setCommunicator()` basically creates two parallelComm instances, while 'assembleComms()' processes meshblock information and then creates sub comms. The meshblock information that is used by 'assembleComms()' seems to be created inside the Tioga function 'register_unstructured_grid()', which is called  by 'TiogaBlock::register_block_ngp()'. So the 'assembleComms()' call quite possibly has to happen after that is called.